src/swsssdk/scripts/sonic-db-cli

```diff
@@ -7,6 +7,14 @@ import argparse
 from multiprocessing import Pool
 from functools import partial
 
+def print_response(resp):
```

Comment:
do we need add similar changes to other python script ? Or only sonic-db-cli uses py3 today in SONiC ?

Reply:
Those who are using the py3 version of py-swsssdk need to take care of decoding. sonic-db-cli happens to be one user. ZTP will be the other user of py3-swsssdk and in my yet to be raised PR's, I take of that as well.